tools/walletextension/storage/sqlite.go

```diff
@@ -0,0 +1,117 @@
+package storage
+
+import (
+	"database/sql"
+	"encoding/hex"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/crypto/ecies"
+	_ "github.com/mattn/go-sqlite3" // sqlite driver for sql.Open()
+	"github.com/obscuronet/go-obscuro/go/rpc"
+)
+
+type SqliteDatabase struct {
+	db *sql.DB
+}
+
+func NewSqliteDatabase(dbName string) (*SqliteDatabase, error) {
+	db, err := sql.Open("sqlite3", dbName)
+	if err != nil {
+		fmt.Println("Error opening datbaase: ", err)
+		return nil, err
+	}
+
+	_, err = db.Exec(`CREATE TABLE IF NOT EXISTS viewingkeys (
+		user_id binary(32),
```

Comment:
I think these are the actual sizes
		user_id binary(32),
		account_address binary(20),
		private_key binary(32),
		signed_key binary(65)

Reply:
changed